index.js

```diff
@@ -12,19 +13,24 @@
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
 // console.log("%cQuestion 2", "color: red") 
-
+//  const name = "Matthew Thomas";
 // console.log(name)
-// => "Raffy"
+// // => "Raffy"
 // name = "Not Raffy" 
-// => TypeError
+// // => TypeError
 // console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
-
+function drinkWater(currentThirstLevel){
+console.log("Man I sure am thirsty")
+currentThirstLevel -= 1
+console.log ("Ahh that hits the spot")
+return currentThirstLevel;
+}
```

Comment:
great! line 30 could also be currentThirstLevel--
you could also combine lines 30 and 32 and just do 
return currentThirstLevel - 1